app/controllers/staff/events_controller.rb

```diff
@@ -18,6 +18,28 @@ def show
   def speaker_emails
   end
 
+  def update_speaker_emails
+    authorize_update
+    if @event.update(params.require(:event).permit(:accept, :reject, :waitlist))
+      flash[:info] = "Your speaker email templates were updated."
+      redirect_to event_staff_speaker_email_notifications_path
+    else
+      flash[:danger] = "There was a problem saving your email templates; please review the form for issues and try again."
+      render :speaker_email_notifications
+    end
+  end
+
+  def remove_speaker_email_template
+    authorize_update
+    if @event.update_attribute(params[:type].to_sym, "")
```

Comment:
I read a little further and saw the types defined on the speaker_email_template model. Referencing that would be better than doing it inline in the controller. I'll leave another comment below, but the types defined in SpeakerEmailTemplate can live in the Event model.